mmv1/third_party/terraform/services/compute/resource_compute_network_firewall_policy_rule_test.go.tmpl

```diff
@@ -149,6 +147,185 @@ func TestAccComputeNetworkFirewallPolicyRule_securityProfileGroup_update(t *test
   })
 }
 
+func TestAccComputeNetworkFirewallPolicyRule_secureTags(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"project_name":  envvar.GetTestProjectFromEnv(),
+		"service_acct":  envvar.GetTestServiceAccountFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeNetworkFirewallPolicyRuleDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeNetworkFirewallPolicyRule_secureTags(context),
+			},
+			{
+				ResourceName:      "google_compute_network_firewall_policy_rule.primary",
+				ImportState:       true,
+				ImportStateVerify: true,
+        // Referencing using ID causes import to fail
+				ImportStateVerifyIgnore: []string{"firewall_policy"},
+			},
+			{
+				Config: testAccComputeNetworkFirewallPolicyRule_secureTagsUpdate(context),
+			},
+			{
+				ResourceName:      "google_compute_network_firewall_policy_rule.primary",
+				ImportState:       true,
+				ImportStateVerify: true,
+        // Referencing using ID causes import to fail
+				ImportStateVerifyIgnore: []string{"firewall_policy"},
+			},
+		},
+	})
+}
+
+func testAccComputeNetworkFirewallPolicyRule_secureTags(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_network_security_address_group" "basic_global_networksecurity_address_group" {
+  name        = "tf-test-address-%{random_suffix}"
+  parent      = "projects/%{project_name}"
+  description = "Sample global networksecurity_address_group"
+  location    = "global"
+  items       = ["208.80.154.224/32"]
+  type        = "IPV4"
+  capacity    = 100
+}
+
+resource "google_compute_network_firewall_policy" "basic_network_firewall_policy" {
+  name        = "tf-test-policy-%{random_suffix}"
+  description = "Sample global network firewall policy"
+  project     = "%{project_name}"
+}
+
+resource "google_compute_network_firewall_policy_rule" "primary" {
+  action                  = "allow"
+  description             = "This is a simple rule description"
+  direction               = "INGRESS"
+  disabled                = false
+  enable_logging          = true
+  firewall_policy         = google_compute_network_firewall_policy.basic_network_firewall_policy.id
```

Comment:
```suggestion
  firewall_policy         = google_compute_network_firewall_policy.basic_network_firewall_policy.id
  project                      = "projects/%{project_name}"
```

This would make the test create the resource with the long-form project id. See below.